### PR TITLE
feat: use gosec severities

### PR DESCRIPTION
**Pull request by ldez**
gosec has severities, then we can use them as value.

### CODE REVIEW

**Review by bombsimon — APPROVED** [review]

File: pkg/golinters/gosec.go [hunk 1]
```diff
@@ -149,6 +150,19 @@ func toGosecConfig(settings *config.GoSecSettings) gosec.Config {
...
 
+func convertScoreToString(score issue.Score) string {
+	switch score {
+	case issue.Low:
+		return "low"
+	case issue.Medium:
+		return "medium"
+	case issue.High:
+		return "high"
+	default:
```
Comment (Antonboom) [inline pkg/golinters/gosec.go:161]: enable in project `exhaustive` linter? 🤔

**Review by Antonboom — COMMENTED** [review]
👍

File: pkg/golinters/gosec.go (hunk 1 above)
Comment (ldez) [reply pkg/golinters/gosec.go:161]: It's a false positive producer, I'm sure that is a good idea.

Comment (Antonboom) [reply pkg/golinters/gosec.go:161]: not sure* 
? :)

**Pull request merged**